couchdb-exporter.go

```diff
@@ -217,6 +214,9 @@ func ofString(i string) *string {
 }
 
 func main() {
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
```

Comment:
Change this one to be consistent with the [README](https://github.com/gesellix/couchdb-prometheus-exporter?tab=readme-ov-file#logging).